### PR TITLE
Issue #3314228: entityreference now in core

### PR DESCRIPTION
**Pull request by herbdool**
Fixes https://www.drupal.org/project/backdrop_upgrade_status/issues/3314228

### CODE REVIEW

File: backdrop_upgrade_status.core.inc [hunk 1]
```diff
@@ -246,6 +246,15 @@ function _backdrop_upgrade_status_backdrop_core(&$data, $project) {
       $data[$project]['in_core_upgrade_status'] = t('Leave enabled.');
       break;
 
+    case 'entityreference':
+      $data[$project]['in_core_since'] = '1.x';
+      $data[$project]['in_core_note'] = t('Moved into core in v1.23.');
+      $data[$project]['in_core_warning'] = t('See <a href="@entityreference">the documentation</a> for more information.', array(
```
Comment (jenlampton) [inline backdrop_upgrade_status.core.inc:252]: We should remove this warning (the docs only apply to people who were using entityreference on their Backdrop sites), and instead add the same info as Link module:
```
      $data[$project]['in_core_complete'] = TRUE;
      $data[$project]['in_core_upgrade_status'] = t('Leave enabled.');

```

edit: On second thought... is it completely in core? If not, we should state exactly what is missing in a warning, similar to menu_block or media.

Comment (herbdool) [reply backdrop_upgrade_status.core.inc:252]: @jenlampton they're the same functionality.

Comment (herbdool) [reply backdrop_upgrade_status.core.inc:252]: @jenlampton I was following the pattern like this one:

```
case 'entitycache':
      $data[$project]['in_core_since'] = '1.x';
      $data[$project]['in_core_note'] = t('Integrated into core Entity module in v1.16.');
      $data[$project]['in_core_warning'] = t('See <a href="@entitycache">the documentation</a> for more information.', array(
        '@entitycache' => 'https://docs.backdropcms.org/change-records/entity-caching-added-to-core',
      ));
      $data[$project]['in_core_upgrade_status'] = t('Leave enabled.');
```

If we remove the warning from one, why not the others?

Comment (laryn) [reply backdrop_upgrade_status.core.inc:252]: @herbdool @jenlampton 
Can we split the difference (and open a new issue on whether to remove the warning on these types of cases)? Maybe inserting `Leave enabled.` in front of the link to the more detailed change record info?

Comment (jenlampton) [reply backdrop_upgrade_status.core.inc:252]: We don't need a warning if there's no differences that could introduce a risk. Entity cache had a rewrite introducing some potentially dangerous differences, hence the warning. 

We can definitely open an issue to more clearly discuss and define when a warning is necessary. My original thought was "only when people need to be warned" about the change. We could certainly add some other kind of notice with "extra information" that isn't a warning :)

**Comment by jenlampton** [comment]
I've added a commit with the following:
```php
    case 'entityreference':
      $data[$project]['in_core_since'] = '1.x';
      $data[$project]['in_core_note'] = t('Moved into core in v1.23.');
      $data[$project]['in_core_complete'] = TRUE;
      $data[$project]['in_core_upgrade_status'] = t('Leave enabled.');
      break;
```
I credietd @herbdool .

**Pull request closed without merging**